modelservers/resnet50/README.md

```diff
@@ -0,0 +1,99 @@
+# Create a pretrained ResNet50 model server and run a load test
+
+#### 1. Create the model
+
+##### Using a Spell Run
+1. Create a run with downloads the model `spell run python make_resnet50.py`
+2. Create a Spell model from the H5 model file using `spell model create --file modelservers/resnet50/model.h5:model.h5 resnet50 runs/<RUN_ID>`
+
+##### Uploading the model
+1. Locally run `python make_resnet50.py`
```

Comment:
nit can we name it save_resnet50.py or something like that, this makes me think its training